### ISSUE TITLE
Interface looks totally different

### ISSUE THREAD

**Issue by rowild** [changeweb/Unifiedtransform#11]
When starting the app, I get a totally different, very rudimentary view of the backend. It seems it is only the route view without any layout. Are there any additional steps to get the backend dashboard running? Some gulp or webpack probably?

**Comment by changeweb** [comment]
Can you share a screenshot?

**Comment by rowild** [comment]
Yes, sure!

When I go to the ./public/ route, the standard Laravel 5.5 layoutis shown (screenshot 1). When I login, then a simple create route is shown within the standard Laravel layout (screenshot 2).

![screenshot_1](https://user-images.githubusercontent.com/213803/54610459-5e859880-4a55-11e9-831e-f3db1fd8e339.png)
![screenshot_2](https://user-images.githubusercontent.com/213803/54610460-5e859880-4a55-11e9-8d05-e840c212d4e0.png)

**Comment by norbybaru** [comment]
@rowild you will need to create a school and assign an admin user to the school. Once that is done, you will need to login with that school admin user to manage that school.

Otherwise when you run the seeders there should be a user assign admin role to a school. Login as that user to manage a school.

**Comment by rowild** [comment]
Thanks for the hint! 
But - how should one know that?? That belongs into the documentation!

**Comment by changeweb** [comment]
@rowild You are right. I just haven't got enough time to document these. I will try to create a doc. BTW you can make a pull request with some doc in readme.md if you want. Any community help is appreciated.

**Comment by changeweb** [comment]
@rowild I added a basic guide in readme.md regarding this issue. You can check that out

**Comment by rowild** [comment]
@changeweb This is a lot better! Thank you! 

Eventually, there is still need for a more detailed instruction, like how to use seeds or - even more important - how to use the system without fake data. I can help with documentation, if that is ok with you, but first I need to get through all the other errors...

**Comment by changeweb** [comment]
@rowild Sure, any help is appreciated. I am sorry for these bugs. Actually the system worked properly. After that I made some changes and then went busy with my other works. So I forgot about most of the changes and did not test them before creating this repository.

Solving your issues are fixing these problems. Thank you for your effort!
This is why I love community to make a product better.  

#14 I fixed registering issue. Please check that out.

**Comment by changeweb** [comment]
I've added a basic instruction about how to use system without fake data. You can check it out.

**Comment by norbybaru** [comment]
@changeweb this is awesome. I will be contributing as much as I can. I don't know want your process for contributing to the project. The major issue first will be to add test coverage before doing any new features or refactoring otherwise someone might break one feature. If you keen for a chat we can make it happen.

**Comment by rowild** [comment]
Tests would be awesome!
@changeweb This issue is solved technically, which means that, in my opinion, this issue could be closed. 

But visually some improvement is needed, right? Do you intend to create a different entry page into the system then the regular Laravel layout? What are your ideas? Maybe I could help here a bit?
A new issue for discussing interface would be better, though...

**Comment by rowild** [comment]
@norbybaru Could you open new issue for the testing topic? Then I could close this issue here.

**Comment by changeweb** [comment]
@norbybaru Feel free to create working tests. Create a new issue regarding this.
@rowild Sure you can close this and create a new issue on layout design improvement.
